packages/terra-horizontal-divider/package.json

```diff
@@ -0,0 +1,45 @@
+{
+  "name": "terra-horizontal-divider",
+  "main": "lib/HorizontalDivider.js",
+  "version": "0.0.0",
+  "description": "The terra-horizontal-divider component is a component that is used to separate content on a layout.",
```

Comment:
description is a bit redundant, I'd remove one of the 'component's from it such as the the second one and say 'component is used to separate'

Reply:
Updated https://github.com/cerner/terra-core/pull/962/commits/9f8a5deba40296bf91341b6ff1d0d56f4bae16ac#diff-3fe4bec3374f1238c4e258d3c3e199feR4